E/EinsteinToolkit/build_tarballs.jl

```diff
@@ -0,0 +1,238 @@
+
+    # GitSource("https://github.com/EinsteinToolkit/ExternalLibraries-BLAS", "93bed30e19f4f09840622d223578aa78639988fe"),
+    GitSource("https://github.com/EinsteinToolkit/ExternalLibraries-FFTW3", "b259730780518efa6333550963c40cee3455dd2c"),
+    GitSource("https://github.com/EinsteinToolkit/ExternalLibraries-GSL", "827085fe750ebfc7359f083614a124915883ca5b"),
+    GitSource("https://github.com/EinsteinToolkit/ExternalLibraries-HDF5", "eeb204b3db68212286641cfe03e7184bcada23f0"),
+    # GitSource("https://github.com/EinsteinToolkit/ExternalLibraries-LAPACK", "de8ea278756d69b723141ffeff6985a9ebe14d1e"),
+    GitSource("https://github.com/EinsteinToolkit/ExternalLibraries-MPI", "8f1b760e35aa8815852e55cdb41779f4583da28f"),
+    GitSource("https://github.com/EinsteinToolkit/ExternalLibraries-OpenBLAS", "1acebb973c547dc8d1c189a652c88c058b8be486"),
+    GitSource("https://github.com/EinsteinToolkit/ExternalLibraries-hwloc", "3fb3343feae2c72cb109dbdeb26db280bf731c70"),
+    GitSource("https://github.com/EinsteinToolkit/ExternalLibraries-libjpeg", "d5c1c5510d18759d7604e5d0bde0e8fd400e664c"),
+    GitSource("https://github.com/EinsteinToolkit/ExternalLibraries-pthreads", "554c0bbbf54cb3ce3157045f24cfd8af71f27054"),
+    GitSource("https://github.com/EinsteinToolkit/ExternalLibraries-zlib", "d15d828036caf2b45a76ea977c1ab47d5658626e"),
+
+    GitSource("https://github.com/GRHayL/GRHayL.git", "6853fe9ae29276621746325d125fa6239460921d"),
+    GitSource("https://github.com/GRHayL/GRHayLET.git", "0b1170359736408423e35674713c1d93a8ac1586"),
+    GitSource("https://github.com/dboyer7/TOVola.git", "7cf7799ac9746380dffcad29f07a894e992776c1"),
+    GitSource("https://github.com/ianhinder/Kranc.git", "b4b2b40103a706a29f8f6b3910110a30afff75aa"),
+    GitSource("https://bitbucket.org/llamacode/llama.git", "3733ef58dca5906abbfb3cbffcc19131048f7e71"),
+    GitSource("https://bitbucket.org/einsteintoolkit/mclachlan.git", "46157bbd3a716dc36c31fde08b1eaea6cabb1ca4"),
+    GitSource("https://bitbucket.org/cactuscode/numerical.git", "535f5975d3ee289400c10c7d761035c63a594a87"),
+    GitSource("https://bitbucket.org/einsteintoolkit/pittnullcode.git", "bc662c6dfde296b8a3f2c32c825bd28da0efaded"),
+    GitSource("https://bitbucket.org/canuda/lean_public.git", "36c1afcfc6305dea54cae62dd3193567e330bb52"),
+    GitSource("https://bitbucket.org/canuda/Proca.git", "79d9ee27a2035e07fa28f67d9fef2bbc6a746912"),
+    GitSource("https://bitbucket.org/canuda/Scalar.git", "06b6062740a98f4ec5cfbe8b7d70a165ec87eeda"),
+    GitSource("https://github.com/wofti/CactusSgrid.git", "c88af8a391c0c7e235aac0750d3529912d2db0ba"),
+    GitSource("https://bitbucket.org/zach_etienne/wvuthorns.git", "7802e89105d8b077e848e62a2e10ed50a79ed8cd"),
+    GitSource("https://bitbucket.org/zach_etienne/wvuthorns_diagnostics.git", "378d2a0e90e37b3d9ac477488ccb370bc0ae22fd"),
+
+    DirectorySource("bundled"),
+]
```

Comment:
Any reason why these aren't separate packages besides the fact they're _a lot_ (which frankly I think would be a good answer here)?